assert/assertions.go

```diff
@@ -1109,6 +1109,27 @@ func WithinDuration(t TestingT, expected, actual time.Time, delta time.Duration,
 	return true
 }
 
+// WithinTimeRange asserts that a time is within a time range (inclusive).
+//
+//   assert.WithinTimeRange(t, time.Now(), time.Now(), time.Now())
```

Comment:
I'm of the opinion that this example isn't the most intuitive. Would it make more sense if we changed it to something like:
```
assert.WithinTimeRange(t, time.Now(), time.Now().Sub(time.Second()), time.Now().Add(time.Second()))
```

If you can think of a simpler example that would explicitly demonstrate the differences between start and end, that would be awesome :)

Reply:
Yeah, I'll think about it a bit and have a look at other examples to see if there's a better way to present it.

Reply:
Changed to `assert.WithinTimeRange(t, time.Now(), time.Now().Add(-time.Second), time.Now().Add(time.Second))`.